### PR TITLE
Define a goal for the specs

### PR DESCRIPTION
**Pull request by jomuel**
(Up for discussion)

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -2,7 +2,19 @@ Raiden Network Specification
 =================================
 
 
-This is a tentative specification for the `Raiden Network <http://raiden.network/>`_. It's under development and will be further refined in subsequent iterations.
+This is the specification of the `Raiden Network <http://raiden.network/>`_.
+
+Its goal is to provide an exact description of the Raiden protocol. So the structure of
+a mediated transfer, message formats and such are to be found here, while implementation
```
Comment (rakanalh) [inline index.rst]: What does `and such` mean in this case? do we have anything else that describes the protocol other than messages?

Comment (rakanalh) [inline index.rst]:
```suggestion
a mediated transfer, message formats and such are to be found here. Implementation-specific
```

File: index.rst [hunk 2]
```diff
@@ -2,7 +2,19 @@ Raiden Network Specification
...
+specific parts of the Raiden client, which is the protocol's reference implementation, are
+documented in the `docs <https://raiden-network.readthedocs.io/en/stable/index.html>`
+instead.
+
+Smart contracts and services, however, are specified in greater detail here since every possible
```
Comment (rakanalh) [inline index.rst]:
```suggestion
Smart contracts and services are also specified in greater detail here since every possible
```

File: index.rst [hunk 3]
```diff
@@ -2,7 +2,19 @@ Raiden Network Specification
...
+implementation of Raiden needs to be developed against them. This makes it necessary, for example,
+to specify the API endpoints for the services, whereas the API of the Raiden client is not documented
+here but in the Raiden docs.
+
```
Comment (rakanalh) [inline index.rst]: `This makes it necessary, for example,
to specify the API endpoints for the services, whereas the API of the Raiden client is not documented
here but in the Raiden docs.` 

Sounds like a justification, please remove.

File: index.rst [hunk 4]
```diff
@@ -2,7 +2,17 @@ Raiden Network Specification
 =================================
 
 
-This is a tentative specification for the `Raiden Network <http://raiden.network/>`_. It's under development and will be further refined in subsequent iterations.
+This is the specification of the `Raiden Network <http://raiden.network/>`_. Its goal is to provide
+an exact description of the Raiden protocol.
+
+For the Raiden client (Raiden node), we will thus describe the format of messages it sends to other
```
Comment (pirapira) [inline index.rst]: Let's talk about plural "Raiden clients" instead of "the Raiden client".  The point is then clearer.

Comment (jomuel) [reply index.rst]: Good idea.

**Pull request merged**